src/equiv.js

```diff
@@ -126,6 +126,48 @@ QUnit.equiv = (function() {
 					parentsB.pop();
 					return true;
 				},
+				
+				"set": function( b, a ) {
+					var aArray, bArray;
+					
+					// b could be an object literal here
```

Comment:
I was actually hesitant to add this in, but I saw that the "array" case had this liner. I didn't quite understand why "array" had it, but I thought there's no harm checking it other than slight inefficiency. If it's indeed useless, I can remove it.